cmake/Dependencies.cmake

```diff
@@ -41,7 +47,6 @@ message(STATUS "soloud NOT FOUND, fetching from source!")
 FetchContent_Declare(
         soloud
         GIT_REPOSITORY https://github.com/onon1101/soloud.git
-        # GIT_TAG noinst
         GIT_TAG onon1101-patch-2
```

Comment:
HASH?

Reply:
![image](https://github.com/NTUT-FUCK-PTSD/Fuck-PTSD/assets/98307453/c3cbc171-8807-481e-a9f3-b4178f1cb5b5)

Reply:
bcz the soloud i made was not release

Reply:
I'm tired to do that